### PR TITLE
Documentation: Add an example scenario that shows outer-delegation

### PR DESCRIPTION
**Pull request by niley7464**
This patch adds an example scenario that shows outer-delegation which MLAgent service handles from the other package.

### CODE REVIEW

File: documentation/getting_started.md [hunk 1]
```diff
@@ -730,3 +729,28 @@ To change the screen to the demonstrate the scenario, 1) open the modal navigati
...
+```
+
+We are providing a simple example application that demonstrates this scenario. This example sends an input prompt to the MLAgent
+and retrieves the generated output text using the llama2 model.
+
+To receive auto-generated text by llama2 model, 1) enter the beginning of the sentence and 2) press the `Run llama2` button.
+
+![Android example](img/A-34-arm64_LlamaEx0.jpeg)
+
+If you want to build this example, you can follow [the guide](https://github.com/nnstreamer/nnstreamer-example/tree/doc/android/kotlin_app#android-application-examples-using-mlops-solution).
```
Comment (niley7464) [inline documentation/getting_started.md:756]: This guide is under revewing
https://github.com/nnstreamer/nnstreamer-example/pull/362

File: documentation/getting_started.md [hunk 2]
```diff
@@ -535,18 +535,17 @@ $ mv model.bin nnstreamer-android/ml_inference_offloading/src/main/assets/models
 $ mv tok4096.bin nnstreamer-android/ml_inference_offloading/src/main/assets/models/tokenizer.bin
 ```
 
-##### Building the llama2.c for Android
-
-To build application with llama2.c, you need to clone this [repository](https://github.com/nnsuite/llama2.c).
+We provide example llama2.c [configuration file](/documentation/example_conf/llama2c.conf).
```
Comment (wooksong) [inline documentation/getting_started.md:538]:
```suggestion
We provide llama2.c as an example [configuration file](/documentation/example_conf/llama2c.conf).
```

File: documentation/getting_started.md [hunk 3]
```diff
@@ -535,18 +535,17 @@ $ mv model.bin nnstreamer-android/ml_inference_offloading/src/main/assets/models
...
+To use it, copy the configuration file to assets repository.
```
Comment (wooksong) [inline documentation/getting_started.md:539]:
```suggestion
To use it, copy the configuration file to the assets directory.
```

File: documentation/getting_started.md [hunk 4]
```diff
@@ -535,18 +535,17 @@ $ mv model.bin nnstreamer-android/ml_inference_offloading/src/main/assets/models
...
-
-$ ls externals/llama2.c/
-...omitted...
+$ cp documentation/example_conf/llama2c.conf ml_inference_offloading/src/main/assets/models
 ```
 
-Then, set the directory where the llama2.c placed to the [gradle.properties](/gradle.properties).
+##### Building the llama2.c for Android
+
+To build application with llama2.c, you need to enable llama2c option in the [gradle.properties](/gradle.properties).
```
Comment (wooksong) [inline documentation/getting_started.md:547]:
```suggestion
To build the application with llama2.c, you need to indicate the directory where llama2.c is placed via the ```dir.llama2c``` property in the [gradle.properties](/gradle.properties).
```

File: documentation/getting_started.md [hunk 5]
```diff
@@ -730,3 +729,28 @@ To change the screen to the demonstrate the scenario, 1) open the modal navigati
 After changing the screen, you can see the **Camera Preview** and the real-time object classification result above the preview box as shown below.
 
 ![Android example](img/A-34-arm64_VisionEx0.png)
+
+#### Handling ML Computation Delegation Requests from the Android Components in the Other package
+
+The MLAgent can handle the ML Computation from the other package. The following code should be added to the other package's `AndroidManifest`
```
Comment (wooksong) [inline documentation/getting_started.md:735]:
```suggestion
The MLAgent can handle the delegation requests from the other package. The following code should be added to the other package's `AndroidManifest.xml`.
```

**Comment by niley7464** [comment]
@wooksong 
All comments have been reflected
Thank you :)

**Review by wooksong — APPROVED** [review]

**Pull request merged**